sql/2019/accessibility/09_06.sql

```diff
@@ -22,7 +22,8 @@ FROM (
     client,
     total,
     page,
-    id)
+    id
+)
```

Comment:
Unfortunatelty v2 doesn't allow trailing closing brackets. But think that's OK as more clear now when line 8 closes.